AddOns/BasicFinancials/app/src/codeunit/BasicFinancialsMgmt.Codeunit.al

```diff
@@ -0,0 +1,121 @@
+codeunit 20601 "Basic Financials Mgmt BF"
+{
+    Access = Internal;
+
+    var
+        AzureADLicensing: codeunit "Azure AD Licensing";
+        NotSupportedLicenseErr: Label 'The Basic Financials Extension can only be deployed, when at least one user has been assigned to a Basic Financials license';
```

Comment:
To deploy the Basic Financials extension the Basic Financials license must be assigned to at least one user.